app/services/doc_auth/lexis_nexis/requests/true_id_request.rb

```diff
@@ -23,6 +23,12 @@ def initialize(
           @liveness_checking_required = liveness_checking_required
         end
 
+        def request_context
+          {
+            workflow: workflow,
```

Comment:
May be we should include, reference id here, so we always have a reference id no matter what response we got from vendor, even no response.